index.html

```diff
@@ -1,4 +1,4 @@
-<html>
+<html style="zoom: 85%;">
```

Comment:
I don't think `zoom` attribute is a good practice in this case.

Btw, if you really want/need to apply `zoom`, I think we should avoid inline CSS styles. Try moving it to `index.scss` `body` class.